cookbooks/bcpc-hadoop/attributes/monitoring.rb

```diff
@@ -0,0 +1,5 @@
+default["bcpc"]["hadoop"]["zabbix"]["history_days"] = 1
+default["bcpc"]["hadoop"]["zabbix"]["trend_days"] = 15
+default["bcpc"]["hadoop"]["zabbix"]["cron_check_time"] = 240
```

Comment:
Put back the `cron_check` code; however keeping it commented since it would take some time to properly understand that function and implement/test it. Can be done in a later update to the work under this PR